website/static/js/fangorn.js

```diff
@@ -420,11 +489,12 @@ function _fangornDropzoneError(treebeard, file, message) {
         }
         if (child.data.tmpID === file.tmpID) {
             item = child;
-            treebeard.deleteNode(parent.id, item.id);
+            tb.deleteNode(parent.id, item.id);
         }
     }
     $osf.growl('Error', msgText);
-    treebeard.options.uploadInProgress = false;
+    tb.options.uploadInProgress = false;
```

Comment:
I'm not exactly sure if this is correct. It's possible to have multiple uploads in progress at the same time, so I think we shouldn't set `uploadInProgress` to `false` when a single upload errors (or completes). It seems like we should only set this to `false` when all uploads have finished. But I'm not sure exactly how this flag is used, so maybe this is expected.